scripts/templates/create-package/EmptyPackageJson.mustache

```diff
@@ -20,7 +20,7 @@
     "start": "npx just-scripts dev",
     "start-test": "npx just-scripts jest-watch",
     "update-snapshots": "npx just-scripts jest -u",
-    "just": "npx just-scripts",
+    "just": "npx just-scripts"
```

Comment:
Thanks! Re: your comment in the PR description about making our JSON reading more robust, if you wanted you could use `jju` (already a dep of the scripts package), which appears to handle trailing commas. What would be even better is just adding a test for the create-package script, but that's certainly not something you need to do right now.

Reply:
> What would be even better is just adding a test for the create-package script, but that's certainly not something you need to do right now.

Agreed. Creating an issue for myself as I'll be on shield starting next week -- https://github.com/OfficeDev/office-ui-fabric-react/issues/9494.